### ISSUE TITLE
loss_seg

### ISSUE THREAD

**Issue by xpp-hhh** [chentao2016/KTSE#4]
loss_inter = torch.sum((cam_max_2nd - complex_cam_max.detach()) * mask_2ndmax) / (torch.sum(mask_2ndmax) + 1e-5)
你好，我发现在这里打印输出的loss是负值，这里计算为什么不用最大的减去第二大的cam 这里有什么设计意图吗？

**Comment by chentao2016** [comment]
因为需要最小化损失，用最大的减去第二大的cam的话，需要最大化损失。不过负的损失确实不太好，可以进行改进。

**Comment by xpp-hhh** [comment]
> 因为需要最小化损失，用最大的减去第二大的cam的话，需要最大化损失。不过负的损失确实不太好，可以进行改进。

好的，感谢
